dot/rpc/subscription/websocket.go

```diff
@@ -265,14 +271,32 @@ func (c *WSConn) initAllBlocksListerner(reqID float64, _ interface{}) (Listener,
 }
 
 func (c *WSConn) initExtrinsicWatch(reqID float64, params interface{}) (Listener, error) {
-	pA := params.([]interface{})
+	var encodedExtrinsic string
 
-	if len(pA) != 1 {
-		return nil, errors.New("expecting only one parameter")
+	switch encodedHex := params.(type) {
+	case []string:
+		if len(encodedHex) != 1 {
+			return nil, fmt.Errorf("%w: %d, expected 1 param", errUnexpectedParamLen, len(encodedHex))
```

Comment:
Nit: Should we be more clear on what we got vs what was expected? Something like fmt.Errorf("%w: expected 1 param, got %d", errUnexpectedParamLen, len(encodedHex)).